DragonFruit.Six.API/Data/Extensions/AccountInfoExtensions.cs

```diff
@@ -27,5 +27,17 @@ public static class AccountInfoExtensions
             var request = new AccountInfoRequest(platform, lookupMethod, queries);
             return client.Perform<JObject>(request, token).DeserializeAccountInfo();
         }
+
+        /// <summary>
+        /// Get a user's account info by name
```

Comment:
these can be changed to use a `see` tag, but that can be addressed at a later date